includes/orders/classes/class-order.php

```diff
@@ -275,12 +275,7 @@ public function __get( $key = '' ) {
 				'order'         => 'ASC',
 			) );
 		} elseif ( 'items' === $key && null === $this->items ) {
```

Comment:
Since the `get_items` method does the `if ( null === $this->items ) {` check, we can probably just simplify this to not check it.

```suggestion
		} elseif ( 'items' === $key ) {
```